app/controllers/admin/userManagement.py

```diff
@@ -48,43 +50,32 @@ def manageUsers():
         flash(user.firstName + " " + user.lastName + " is no longer a CELTS Student Staff", 'success')
     return ("success")
 
-@admin_bp.route('/addProgramManagers', methods=['POST'])
-def addProgramManagers():
-    eventData = request.form
-    try:
-        return addProgramManager(eventData['username'],int(eventData['programID']))
-    except Exception as e:
-        print(e)
-        flash('Error while trying to add a manager.','warning')
-        abort(500,"'Error while trying to add a manager.'")
 
-@admin_bp.route('/removeProgramManagers', methods=['POST'])
-def removeProgramManagers():
-    eventData = request.form
-    try:
-        return removeProgramManager(eventData['username'],int(eventData['programID']))
-    except Exception as e:
-        print(e)
-        flash('Error while removing a manager.','warning')
-        abort(500,"Error while trying to remove a manager.")
+@admin_bp.route('/deleteProgramFile', methods=['POST'])
+def deleteProgramFile():
+    programFile=FileHandler(programId=request.form["programID"])
+    programFile.deleteFile(request.form["fileId"])
+    return ""
 
 @admin_bp.route('/admin/updateProgramInfo/<programID>', methods=['POST'])
 def updateProgramInfo(programID):
-    """Grabs info and then outputs it to logic function"""
-    programInfo = request.form # grabs user inputs
     if g.current_user.isCeltsAdmin:
         try:
-            changeProgramInfo(programInfo["programName"],  #calls logic function to add data to database
-                              programInfo["contactEmail"],
-                              programInfo["contactName"],
-                              programInfo["location"],
-                              programID)
+            programInfo = request.form # grabs user inputs
+            uploadedFile = request.files.get('modalProgramImage')
+            changeProgramInfo(programID, uploadedFile, **programInfo) 
 
+            associatedAttachments = list(AttachmentUpload.select().where(AttachmentUpload.program == programID).execute()) 
+           
+            filePaths = FileHandler(programId=programID).retrievePath(associatedAttachments) 
+           
+          
+            file_paths = {filename: path_info[0] for filename, path_info in filePaths.items()} 
```

Comment:
The `"filePaths" `variable and the `"file_paths" `variable should be both camelcase. The also should have different variable names.

Reply:
Honestly both lines seem to be assigning variables to objects or dictionaries that are not used in the rest of the function. So I decided to scrap them.